### ISSUE TITLE
Add dynamic autopilot config evaluation into the UI

### ISSUE THREAD

**Issue by alexfreska** [SiaFoundation/renterd#1137]
- the UI would send the first request, e.g. for 50 hosts or how many the user wants
- the server responds with "these settings are already good enough for 60 hosts, no recommendation"
- the UI then sends another request in the background with 1.5 times the user hosts but otherwise the same settings
- the server responds with "these settings are good enough for 60 hosts, here is a recommendation to get to 75"
- the UI presents that to the user

Basically the UI would make sure that even if the user settings are satisfiable, the user has a reasonable buffer of hosts to choose from. Maybe not just 50% but 100% even. So that for every host you need in your set, you have 2 to choose from.

https://github.com/SiaFoundation/renterd/pull/995
